test/e2e/cypress/integration/host_details.js

```diff
@@ -0,0 +1,67 @@
+import { selectedHost } from "../fixtures/host-details/selected_host";
+
+context("Host Details", () => {
+  before(() => {
+    cy.resetDatabase();
+    cy.loadScenario("healthy-27-node-SAP-cluster");
+
+    const agents = new Array();
+    agents.push(selectedHost.agentId);
+    cy.task("startAgentHeartbeat", agents);
+
+    cy.visit("/");
+    cy.navigateToItem("Hosts");
+    cy.intercept("GET", `/hosts/${selectedHost.agentId}`).as("getHostDetails");
+    cy.get(`#host-${selectedHost.hostName} > .tn-hostname > a`).click();
+    cy.wait("@getHostDetails");
+    cy.url().should("include", `/hosts/${selectedHost.agentId}`);
+  });
+
+  describe("Detailed view for a specific host should be available", () => {
+    it("should show the host I clicked on in the overview", () => {
+      cy.get(
+        ".border-top > :nth-child(1) > .col-sm-12 > .row > :nth-child(1) > .text-muted"
```

Comment:
I'd consider adding some more specific css class in order to simplify selector. Something like a container `tn-host-detail-container` and then the name in a child element like `tn-hostname`

That would simplify selecting `.tn-host-detail-container > .tn-hostname`

What do you think?